lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -26,6 +26,7 @@ class SelfPacedDateOverrideTest(ModuleStoreTestCase):
     """
 
     def setUp(self):
+        self.reset_setting_cache_variables()
```

Comment:
@adampalay The one this note is attached to.  I don't know if it was on the flaky tests list, but it was definitely flaky.